shared/chat/conversation/new-team/container.js

```diff
@@ -0,0 +1,32 @@
+// @flow
+import NewTeamDialog from './'
+import {connect} from 'react-redux'
+import {compose, withState, withHandlers} from 'recompose'
+import {navigateTo, navigateUp} from '../../../actions/route-tree'
+import {chatTab} from '../../../constants/tabs'
+import {createNewTeam, selectConversation} from '../../../actions/chat/creators'
+import {getSelectedConversation} from '../../../constants/chat'
+
+import type {TypedState} from '../../../constants/reducer'
+import type {ConversationIDKey} from '../../../constants/chat'
+
+const mapStateToProps = (state: TypedState) => ({
+  conversationIDKey: getSelectedConversation(state),
+})
+
+const mapDispatchToProps = (dispatch: Dispatch) => ({
+  navToRootChat: () => dispatch(navigateTo([], [chatTab])),
+  onBack: () => dispatch(navigateUp()),
+  onCreateTeam: (conversationIDKey: ConversationIDKey, name: string) => {
+    dispatch(createNewTeam(conversationIDKey, name))
+    dispatch(selectConversation(null, true))
```

Comment:
Oh, that'd be good!  But the new conversation won't actually exist for another five seconds or so, until Gregor sends out an intro message to the team that creates an inbox entry for it.  And we don't get an RPC notification when the channel's been created.  So it's hard to know when the right time to refresh the inbox and select it would be..